### PR TITLE
Clarify different sendMessage types

### PR DESCRIPTION
**Pull request by dvoytenko**

### CODE REVIEW

**Comment by cramforce** [comment]
LGTM

File: src/service/viewer-impl.js [hunk 1]
```diff
@@ -826,13 +829,17 @@ export class Viewer {
...
+   * this time, this method responds immediately with a Promise that yields
+   * `undefined` value.
    * @param {string} eventType
    * @param {*} data
    * @param {boolean} awaitResponse
    * @return {!Promise<*>|undefined}
    * @private
    */
-  sendMessage_(eventType, data, awaitResponse) {
+  sendMessageUnsafe_(eventType, data, awaitResponse) {
```
Comment (cramforce) [inline src/service/viewer-impl.js]: How about calling this `Unreliable` instead of `Unsafe`. Or maybe `FireAndForget`?

Comment (dvoytenko) [inline src/service/viewer-impl.js]: Like `Unreliable`. Or maybe 'NotGuranteed'?

Comment (cramforce) [inline src/service/viewer-impl.js]: What exactly is not guaranteed? That we send at all?

I think Unreliable is good then.

Comment (dvoytenko) [inline src/service/viewer-impl.js]: Done

**Pull request merged**